buildSrc/src/main/groovy/com/google/firebase/gradle/plugins/ci/device/FirebaseTestServer.groovy

```diff
@@ -68,7 +67,10 @@ class FirebaseTestServer extends TestServer {
         Optional<String> resultsBucket = Optional.ofNullable(System.getenv('FTL_RESULTS_BUCKET')).map(Environment.&expand)
         Optional<String> resultsDir = Optional.ofNullable(System.getenv('FTL_RESULTS_DIR')).map(Environment.&expand)
 
-        List<String> args = ['--results-bucket', resultsBucket.orElse(DEFAULT_BUCKET_NAME)]
+        List<String> args = []
+        if (resultsBucket.isPresent()) {
+            args += ['--results-bucket', resultsBucket.get()]
+        }
         if (resultsDir.isPresent()) {
             args += ['--results-dir', Paths.get(resultsDir.get(), "${project.path}_${random.nextLong()}")]
```

Comment:
`nextLong` can give negative numbers. I don't know if you think that is a problem.

Reply:
It has not been a problem so far, but I would not be opposed to `Math.abs()` on it, but totally optional.